utils/utils.go

```diff
@@ -22,26 +22,18 @@ func OpenInBrowser(url string) error {
 }
 
 func RenderMarkdown(text string) (string, error) {
-	style := "notty"
-	// TODO: make color an input parameter
-	if isColorEnabled() {
-		style = "auto"
-	}
-
 	// Glamour rendering preserves carriage return characters in code blocks, but
 	// we need to ensure that no such characters are present in the output.
 	text = strings.ReplaceAll(text, "\r\n", "\n")
 
-	tr, err := glamour.NewTermRenderer(
-		glamour.WithStandardStyle(style),
-		// glamour.WithBaseURL(""),  // TODO: make configurable
-		// glamour.WithWordWrap(80), // TODO: make configurable
```

Comment:
I feel that it was more clear how to exactly make BaseURL and WordWrap configurable with the old instructions rather than with the new ones. Could we go back to using more verbose syntax?

Reply:
Yeah. I went with the current syntax because it seemed cleaner with the `if` blocks, but I guess it *would* be better to do it the previous way to get the increase in functionality later down the road when the `TODO`s are actually implemented.